src/Model/User.php

```diff
@@ -146,6 +146,15 @@ public static function getSystemAccount()
 		$system['sprvkey'] = $system['uprvkey'] = $system['prvkey'];
 		$system['spubkey'] = $system['upubkey'] = $system['pubkey'];
 		$system['nickname'] = $system['nick'];
+		$system['page-flags'] = User::PAGE_FLAGS_SOAPBOX;
+		$system['account-type'] = $system['contact-type'];
+		$system['guid'] = '';
+		$system['nickname'] = $system['nick'];
+		$system['pubkey'] = $system['pubkey'];
+		$system['locality'] = '';
+		$system['region'] = '';
+		$system['country-name'] = '';
+		$system['net-publish'] = false;
```

Comment:
`$system['nickname'] = $system['nick'];`

Umm. This line exists two times.

Reply:
Better safe than sorry!